arch/arm/core/cortex_m/pm_s2ram.S

```diff
@@ -91,7 +165,7 @@ SECTION_FUNC(TEXT, arch_pm_s2ram_suspend)
 	/* Move system_off back to r0 as return value */
 	mov	r0, r4
 
-	pop	{r4-r12, lr}
+	pop_gprs
```

Comment:
IIUC at this stage, the registers at the top of the stack are the special purpose ones.
`sp` should be shifted first before restoring `gprs`.

Reply:
Special purpose registers are saved in the `_cpu_context` variable and not on the stack, AFAIU?

Reply:
That’s right! my mistake.